cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/validator/AwsRangerAuditRolePermissionValidatorTest.java

```diff
@@ -0,0 +1,90 @@
+package com.sequenceiq.cloudbreak.cloud.aws.validator;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Assert;
+
+import com.sequenceiq.cloudbreak.cloud.model.filesystem.CloudS3View;
+import com.sequenceiq.common.api.cloudstorage.StorageLocationBase;
+import com.sequenceiq.common.model.CloudIdentityType;
+import com.sequenceiq.common.model.CloudStorageCdpService;
+import com.sequenceiq.common.model.FileSystemType;
+
+public class AwsRangerAuditRolePermissionValidatorTest extends AwsIDBrokerMappedRolePermissionValidatorTest {
+    private final AwsRangerAuditRolePermissionValidator awsRangerAuditRolePermissionValidator =
+        new AwsRangerAuditRolePermissionValidator();
+
+    @Override
+    public void testGetUsers() {
+        List<String> expectedUsers = Arrays.asList("kafka", "solr", "knox", "atlas");
```

Comment:
#6700 has been merged, so please free to use `AccountMappingSubject.RANGER_AUDIT_USERS` instead.